src/components/common/scheduleTable.jsx

```diff
@@ -1,45 +1,43 @@
-import React from "react";
-import Trash from "./trash";
-import Leaf from "./leaf";
-import Recycle from "./recycle";
-const ScheduleTable = ({ services, renderWeekOfDay }) => {
-  // const {services,renderWeekOfDay} = props;
-
-   return (
+import React from 'react';
+import Icon from './Icon';
+const ScheduleTable = ({ services, scheduleType, renderWeekOfDay }) => {
+  const trash = scheduleType.trash.toLowerCase();
+  const leaf = scheduleType.leaf.toLowerCase();
+  const recycle = scheduleType.recycle.toLowerCase();
+  return (
     <table className="table table-bordered table-sm">
       <thead>
         <tr>
           <th>Type</th>
-          <th>Image</th>
           <th>Collection Days</th>
           <th>Next Collection</th>
         </tr>
       </thead>
       <tbody>
         {services.map(service => (
           <tr key={service._id}>
-            <td>{service.type}</td>
             <td align="center">
-              {service.type === "Trash" ? (
-                <Trash />
-              ) : service.type === "Leaf" ? (
-                <Leaf />
+              {service.type.toLowerCase() === trash ? (
+                <Icon iconClass="trash" />
+              ) : service.type.toLowerCase() === leaf ? (
+                <Icon iconClass="leaf" />
               ) : (
-                <Recycle />
+                <Icon iconClass="recycle" />
               )}
+              {service.type}
             </td>
             <td>{service.collectionDays}</td>
             <td>
-              {service.type === "Trash"
-                ? renderWeekOfDay(1)
-                : service.type === "Leaf"
-                ? renderWeekOfDay(3)
-                : renderWeekOfDay(2)}
+              {service.type.toLowerCase() === trash
```

Comment:
this can just be 

```jsx
renderWeekOfDay(service.type.toLowerCase());
```

Reply:
i would also name this `renderWeekOfDay` or maybe even better `renderDayofWeek`.

Reply:
name changed to renderDayofWeek and passed value as mention